### ISSUE TITLE
Audio file from nowhere starts playing with all services disabled

### ISSUE THREAD

**Issue by francwalter** [ferdium/ferdium-app#1821]
### Avoid duplicates

- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22bug+%3Abeetle%3A%22) for a bug report similar to mine, in vain

### Ferdium Version

6.7.4

### What Operating System are you using?

macOS

### Operating System Version

Mac OS 10.15.7

### What arch are you using?

x64

### Expected Behavior

No audio should play, when all auto play is disabled.

### Actual Behavior

After a while, sometimes suddenly audio plays in background

### Steps to reproduce

I have since some weeks a very weird issue: on my old MacBook from Mid 2009 with OS 10.15.7 (patched) I have Ferdium 6.7.4 running and sometimes audio files from nowhere are starting to play and loop.

I have several services (WhatsApp, Telegram, Google Keep, Discord, Google Calendar, IRCCloud and some Website services) and sometimes, without any user interaction, weird audio clips are playing. They play in auto-repeat and I dont see from where its coming.
The audios are mostly german, but also english (american english). Could be some youtube clips (in one audio sombody said: "...write it into the comments...").

First I didnt notice where it was coming from, no app running audio on my MacBook, but I found out, when I close Ferdium, the audio stops immediately, so it is related to Ferdium.
So I started to disable one service after the other, but the audio continues with all services disabled. Only when I close Ferdium completely, the audio stops. Or if I kill VTDecoderXPCService (the audio player service, I guess) in the Activities app under Ferdium.

A while I thought WhatsApp (and maybe some weird contact of mine) could have be the culprit, but I am not anymore sure.
If I disable WhatsApp service the audio continues, but maybe Ferdium is caching them and continues play them even the service is disabled.
It is difficult to check which service it could be, if it is a service, because it happens only sometimes.

### Additional information

I asked this already in [Discussions](https://github.com/orgs/ferdium/discussions/1783), but no answer since weeks. I think it is a bug and it could be a security issue.
Then I asked it in a[ german Mac forum](https://www.macuser.de/threads/ferdium-dubiose-audiodateien-starten-unaufgefordert-im-hintergrund.942381/#post-12305577), but there nobody could help neither, so I start this issue.

**Comment by Alphrag** [comment]
Hi @francwalter, sorry about the discussion thread, we don't really get any notifications from there by default unless we set it up, so we completely missed your question (and multiple others).

I have never experienced any problem like this, and I think this might be related to hibernation. By default Ferdium does not visit any website or any places where there would be audio being played, which means this is necessary related to one of the services you use.
Think of a service in Ferdium as a special tab in a browser. If the website you visit has a video automatically playing when you enter the website then the only way to make it stop is either to manually stop it, or to close the tab. In a browser, your tab stay loaded even if you switch somewhere else, which means that if you stopped the video it shouldn't restart playing by itself. In Ferdium the hibernating mode means that some services can be temporarily unloaded, and at some given interval the service will be reloaded to check if there are new notifications. This could then trigger a video being played if the website associated with that service has such a thing.
So I would suggest you verify if the home page of any of the services you have is a website where there is a video automatically playing (the page that is loaded by default from this service, which you can verify by clicking the `Home` button when the navigation bar is active) and whether that service has a hibernation policy. Notice that if you have multiple workspaces you should check all of them in case you have them loaded at all time.

If you don't find the service that way, consider trying to run a different instance of Ferdium (so you should rename/save your profile folder and delete it completely —or part of it— and using the old profile to bring back services without having to reconfigure everything) with only a few basic services and adding the old ones one by one when you are sure the active set is not problematic.

**Comment by francwalter** [comment]
Thank you for your answer @Alphrag !

Since some days it didn't happen again, after I changed my privacy settings in telegram service to allow sending me anything only from my contacts and I guess (really not sure) that this could have been the culprit. 
Together with hibernation, as you suggested, to trigger the audios. 

What was strange, is that the audios continued playing when I disabled all services. I thought disabling a service should end and kill all related activities from this service. Maybe that is not possible? Maybe a bug?
But at least it should not trigger again in a disabled service.
Is it possible that Ferdium continues de-hibernation even on disabled services? 

But it is hard to reproduce because it was happening not that often and without visible rule. 

I will do a reverse test like re enabling in telegram all non-privacy settings (allow messages etc from sender not in my contact list) and check if the audios come again, if yes then I will disable hibernation in telegram and wait if it comes again...
